change/@fluentui-react-charting-23ac9e54-88e2-456f-abde-468868ed7d1a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix for bug 8380",
```

Comment:
Mention the actual issue and fix instead of bug number

Reply:
Done